docs/graphql/core/guides/postgres/import-data.rst

```diff
@@ -0,0 +1,67 @@
+.. meta::
+   :description: import data from csv in postgres
+   :keywords: hasura, docs, postgres, import, data
+
+.. _postgres_import_data_from_csv:
+
+Import data from csv into database
+===================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+-------------
+
+CSV files are plain text files that makes it easy for any developer to create and add data. 
+Also, the data are easy to import into any database. 
+
+Step 1: Create a Postgres instance
```

Comment:
We can skip the creation of a PG instance. We can assume people already have one. The title here can be "Add a table to your PG database"